pallets/ajuna-nft-staking/benchmarking/src/lib.rs

```diff
@@ -101,19 +101,20 @@ fn assert_last_event<T: Config>(avatars_event: Event<T>) {
 
 fn create_creator<T: Config>(reward_item: Option<Vec<u16>>) -> Result<T::AccountId, DispatchError> {
 	let creator = account::<T>("creator");
-	create_contract_collection::<T>(&creator)?; // reserve CONTRACT_COLLECTION
-	create_collections::<T>(&creator, 1)?; // reserve REWARD_COLLECTION
+	create_contract_collection::<T>(&creator).expect("Account should be created"); // reserve CONTRACT_COLLECTION
+	create_collections::<T>(&creator, 1).expect("Account should be created"); // reserve REWARD_COLLECTION
 	if let Some(item_ids) = reward_item {
 		item_ids
 			.into_iter()
-			.try_for_each(|item_id| mint_item::<T>(&creator, REWARD_COLLECTION, item_id))?;
+			.try_for_each(|item_id| mint_item::<T>(&creator, REWARD_COLLECTION, item_id))
+			.expect("Account should be created");
 	}
 	Creator::<T>::put(&creator);
 	Ok(creator)
 }
 
 fn create_contract_collection<T: Config>(creator: &T::AccountId) -> DispatchResult {
-	create_collection::<T>(creator)?;
```

Comment:
This seems to me like it was a search and replace error. No reason to replac the `?` here.

Btw. We could have chosen to return a `Result<(), BenchmarkError>` in the benchmarks, then we can keep the `?`. I have no strong opinion here though.